pynamodb/connection/base.py

```diff
@@ -178,9 +178,10 @@ class Connection(object):
     A higher level abstraction over botocore
     """
 
-    def __init__(self, region=None, host=None, session_cls=None):
+    def __init__(self, region=None, host=None, session_cls=None, retry_enabled=False):
         self._tables = {}
         self.host = host
+        self.retry_enabled = retry_enabled
```

Comment:
we should allow the user to configure the parameters of retry, not enable/disable

to disable the user can pass max_attempts = 1